### PR TITLE
Update common files

### PR DESCRIPTION
**Pull request by julienbourdeau**
:gift:

### CODE REVIEW

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Here are a few tips to increase the likelihood of being merged:
+
+- [ ] Write tests.
+- [ ] Write a [good commit message](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+- [ ] Allow [edits from maintainers](https://blog.github.com/2016-09-07-improving-collaboration-with-forks/).
+
+### API Breaking changes
+
+We care deeply about backward compatibility for our API clients libraries. If it's necessary, we're ready to break backward compatibility but this is should be pretty rare.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: this is should -> this should

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+If you want to make breaking changes, open an issue first to discuss it with maintainers.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: "the" maintainers

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------|
+| ![#050f2c](https://placehold.it/15/050f2c/000000?text=+) Do not merge    | PR should not be merged (decided by maintainers)                                       |
+| ![#ffc168](https://placehold.it/15/ffc168/000000?text=+) WIP             | PR is not ready, no need to look at it (decided by contributors)                       |
+| ![#2ede98](https://placehold.it/15/2ede98/000000?text=+) Ready           | The PR is ready, reviewed, tests are green, if you're brave enough: click merge button |
+| ![#ffc168](https://placehold.it/15/ffc168/000000?text=+) Waiting for API | The feature is implemented but the REST API is not live yet                            |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) Discussion      | We need everyone's opinion on this, please join the conversation and share your ideas  |
+| ![#3369e6](https://placehold.it/15/3369e6/000000?text=+) Support         | User need help but it's not really a bug                                               |
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: "A user" or "User needs"

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Hi there! We're thrilled that you'd like to contribute to this project.
+Your help is essential for keeping it great.
+
+## Before you start coding
+
+### Editing `README.md` and similar files
+
+Note that some files are managed outside this repository and are committed automatically.
+
+`README.md` are is generated automatically from our doc. If you'd like to see more details about something, feel free to open an issue.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: are is -> is
and I would probably start the sentence with "The"

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+`.github` directory is manage in [this repository](https://github.com/algolia/algoliasearch-client-common), any Pull Request there are welcome.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: "is welcome"

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## Opening an issue
+
+Each repository provide a template for issues. Please fill the client and language version and provide a clear description of the problem you're facing. Steps to reproduce or example code (repository, jsfiddle and such) are a big help.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: Each repository provide -> Each repository provides

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+## Submitting a pull request
+
+Keep your changes as focused as possible. If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
```
Comment (rayrutjes) [inline .github/CONTRIBUTING.md]: If there are multiple changes you would like to make that are not dependent upon each other, consider submitting them as separate pull requests.
=> this sentence is unclear

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Comment by julienbourdeau** [comment]
Thanks @rayrutjes for your feedback. I didn't get you were opening a PR on the common so I applied the changes.

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Comment by julienbourdeau** [comment]
The PR was updated, please have a look and merge if no issue were found

**Pull request merged**